.travis.yml

```diff
@@ -33,7 +33,7 @@ cache:
 
 notifications:
```

Comment:
Is notifications section mandatory if we don't want any notifications? Maybe we could remove it at all?